projects/plugins/jetpack/modules/search/instant-search/components/path-breadcrumbs.jsx

```diff
@@ -8,10 +8,16 @@ import './path-breadcrumbs.scss';
 function splitDomainPath( path ) {
 	const splits = path.split( '/' ).filter( piece => piece.length > 0 );
 	splits.shift(); // Removes domain name from splits; e.g. 'jetpack.com'
-	return splits.length === 0 ? [ '/' ] : splits;
+	return splits.length === 0 ? [] : splits;
 }
 
 const PathBreadcrumbs = ( { className, onClick, url } ) => {
+	const breadcrumbPieces = splitDomainPath( url );
+
+	if ( breadcrumbPieces.length < 1 ) {
```

Comment:
More information about conditional rendering: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
Do you think we should still display the `div` wrapper for the breadcrumb even if it is empty @kangzj?

Reply:
> Do you think we should still display the `div` wrapper for the breadcrumb even if it is empty @kangzj?

I thought about this. TBH I'm not sure. I would still display the div IMO as it'll be more consistent code-wise.

Reply:
I wonder if keeping the `div` wrapper without a breadcrumb inside would have any styling consequences? But I do agree that it would be consistent code-wise to keep it there, too. I can make this change if we feel it's the best way forward?

Reply:
I think it'll be fine to keep the div - it wouldn't do anything visually if we don't want so, which also means we could save the code from L15 to L19 😄

Reply:
I definitely don't think we want to render the `<a>` at all if there is nothing in `breadcrumbParts`.

As far as the wrapper div goes, shall we have a look at our other components and see what they do when there's no content?

Reply:
It looks like the `search-result-comments` component only outputs the `<div>` if there are comments.

However, the image `<div>` containers and `<a>`  `jetpack-instant-search__search-result-expanded__image-link` is output even when there is no image to display. 

So it looks like this could go either way?

Reply:
This approach looks reasonable to me. We don't have any breadcrumbs to render, so we don't render anything. Omitting the breadcrumbs div doesn't appear to have any meaningful impact.

The reason we indiscriminately render `jetpack-instant-search__search-result-expanded__image-link` is tech debt. At some point in the past, we used to rely on that empty element to ensure correct styling. I believe that's no longer necessary now that we're using flexbox for the parent element (specifically, `flex-flow: row nowrap`).